schemas/sds/1.3/source-data-stream-1.3.sch

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- This schematron has been created based on the original SCAP 1.3 schematron from NIST -->
+<sch:schema xmlns:sch="http://purl.oclc.org/dsdl/schematron"
+ xmlns:xsl="http://www.w3.org/1999/XSL/Transform" queryBinding="xslt">
+  <sch:ns prefix="ai" uri="http://scap.nist.gov/schema/asset-identification/1.1"/>
+  <sch:ns prefix="arf" uri="http://scap.nist.gov/schema/asset-reporting-format/1.1"/>
+  <sch:ns prefix="cat" uri="urn:oasis:names:tc:entity:xmlns:xml:catalog"/>
+  <sch:ns prefix="cpe-dict" uri="http://cpe.mitre.org/dictionary/2.0"/>
+  <sch:ns prefix="cpe-lang" uri="http://cpe.mitre.org/language/2.0"/>
+  <sch:ns prefix="dc" uri="http://purl.org/dc/elements/1.1/"/>
+  <sch:ns prefix="ds" uri="http://scap.nist.gov/schema/scap/source/1.2"/>
+  <sch:ns prefix="dsig" uri="http://www.w3.org/2000/09/xmldsig#"/>
+  <sch:ns prefix="java" uri="java:gov.nist.secauto.scap.validation.schematron"/>
+  <sch:ns prefix="nvd-config" uri="http://scap.nist.gov/schema/feed/configuration/0.1"/>
+  <sch:ns prefix="ocil" uri="http://scap.nist.gov/schema/ocil/2.0"/>
+  <sch:ns prefix="oval-com" uri="http://oval.mitre.org/XMLSchema/oval-common-5"/>
+  <sch:ns prefix="oval-def" uri="http://oval.mitre.org/XMLSchema/oval-definitions-5"/>
+  <sch:ns prefix="oval-res" uri="http://oval.mitre.org/XMLSchema/oval-results-5"/>
+  <sch:ns prefix="oval-sc" uri="http://oval.mitre.org/XMLSchema/oval-system-characteristics-5"/>
+  <sch:ns prefix="rc" uri="http://scap.nist.gov/schema/reporting-core/1.1"/>
+  <sch:ns prefix="scap" uri="http://scap.nist.gov/schema/scap/source/1.2"/>
+  <sch:ns prefix="scap-con" uri="http://scap.nist.gov/schema/scap/constructs/1.2"/>
+  <sch:ns prefix="tmsad" uri="http://scap.nist.gov/schema/xml-dsig/1.0"/>
+  <sch:ns prefix="xccdf" uri="http://checklists.nist.gov/xccdf/1.2"/>
+  <sch:ns prefix="xcf" uri="nist:scap:xslt:function"/>
+  <sch:ns prefix="xinclude" uri="http://www.w3.org/2001/XInclude"/>
+  <sch:ns prefix="xlink" uri="http://www.w3.org/1999/xlink"/>
+  <sch:ns prefix="xml" uri="http://www.w3.org/XML/1998/namespace"/>
+  <sch:ns prefix="xsd" uri="http://www.w3.org/2001/XMLSchema"/>
+
+  <sch:pattern id="scap-general">
+    <sch:rule id="scap-general-xccdf-benchmark" context="xccdf:Benchmark">
+      <sch:assert id="scap-general-xccdf-benchmark-description" test="xccdf:description">SRC-10-1|xccdf:Benchmark <sch:value-of select="@id"/></sch:assert>
```

Comment:
@ggbecker I think that if an user will see `SRC-10-1` in `oscap` output they won't know what is that. I think that we should try to add some description. Or at least add everywhere `Schematron validation error` or something. What do you think?

Reply:
Does NIST provide a document with the requirements? I think we can at least point to this document and throw the generic "Schematron validation error"

Reply:
We should emit the same message original 2.0 Schematron does. Is it possible to do?

Reply:
Yes, they provide, it's in the same zip from NIST as the schematron file, it's `requirements/scapval-scap-1.3-requirements.xml`

As you can see in the original schematron there are these short messages - they come from there. I have mostly just copied them.

Reply:
we should at least enable distinguishing between warnings and errors